configuration/haproxy.go

```diff
@@ -8,4 +10,29 @@ type HAProxy struct {
 	ReloadCleanupCommand    string
 	ShutdownCommand         string
 	GraceSeconds            int
+	HostnameLabel           *string
+}
+
+// BalancerType indicates whether there bamboo is dealing with external or internal traffic
+type BalancerType string
```

Comment:
Interesting. I didn't know we had plans to do that, but it seems like it would be a great improvement. Do you think we should address that now or in a future PR?